components/blitz/resources/omero/api/IMetadata.ice

```diff
@@ -18,37 +18,212 @@ module omero {
 
     module api {
         /**
-         * See <a href="http://downloads.openmicroscopy.org/latest/omero5.2/api/ome/api/IMetadata.html">IMetadata.html</a>
+         * Provides method to interact with acquisition metadata and
+         * annotations.
          **/
         ["ami", "amd"] interface IMetadata extends ServiceInterface
             {
+                /**
+                 * Loads the <code>logical channels</code> and the acquisition
```

Comment:
If "logical channels" is code should it be CamelCase? or at least not have a space in it?